lib/components/viewers/stop-viewer.js

```diff
@@ -240,35 +342,61 @@ class RouteRow extends Component {
   }
 }
 
+const ONE_HOUR_IN_SECONDS = 3600
+
+function getHomeTime (homeTimezone) {
+  const now = moment()
+  return now.tz(homeTimezone).diff(now.clone().startOf('day'), 'seconds')
+}
+
 // helper method to generate stop time w/ status icon
-function getFormattedStopTime (stopTime, homeTimezone) {
+function getFormattedStopTime (stopTime, homeTimezone, arrivingText = 'Due') {
```

Comment:
Any complex function delaying with displaying time and timezones ought to have tests with all kinds of edge cases. Let's add an issue for this.

Reply:
#63.